.github/workflows/ci.yml

```diff
@@ -18,3 +18,18 @@ jobs:
         fetch-depth: 0
     - run: sudo rm -rf /usr/local/bin/kustomize
     - run: make test
+
+  check-manifest:
+    name: check manifests
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+      id: go
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - run: |
+        make manifests && git diff --exit-code
```

Comment:
In order for this to work properly, I think the commands need to be in individual run: steps. And, I think these can just go between L19 and L20.

So, it would look like this:

```
- run: sudo rm -rf /usr/local/bin/kustomize
- run: make generate && git diff --exit-code
- run: make manifests && git diff --exit-code
- 1run: make test
```

Then you won't need to duplicate the setup.